src/presentation/vendorProfile/vendorProfile.js

```diff
@@ -1,20 +1,156 @@
 import React from 'react'
 import { get } from 'lodash'
+import { Card, CardMedia, CardText, CardTitle } from 'material-ui/Card'
+import RaisedButton from 'material-ui/RaisedButton'
 
-const ForgotPasswordPage = (props) => {
+const defaultImage = 'http://sunfieldfarm.org/wp-content/uploads/2014/02/profile-placeholder.png'
+
+function formatPhoneNumber(s) {
+    const s2 = (`${s}`).replace(/\D/g, '')
+    const m = s2.match(/^(\d{3})(\d{3})(\d{4})$/)
+    return (!m) ? null : `(${m[1]}) ${m[2]}-${m[3]}`
+}
+
+function linkToEmbed(link, type) {
+    if (link) {
+        switch (type) {
+            case 'youtube':
+                const youtubeRegExp = /^.*((youtu.be\/)|(v\/)|(\/u\/\w\/)|(embed\/)|(watch\?))\??v?=?([^#&?]*).*/
+                const youtubeMatch = link.match(youtubeRegExp)
+                const youtubeVideoID = youtubeMatch[7]
+
+                return `https://www.youtube.com/embed/${youtubeVideoID}`
+
+            case 'vimeo':
+                // Source: http://jsbin.com/asuqic/184/edit?html,js,output
+                const vimeoRegExp = /https?:\/\/(?:www\.)?vimeo.com\/(?:channels\/(?:\w+\/)?|groups\/([^/]*)\/videos\/|album\/(\d+)\/video\/|)(\d+)(?:$|\/|\?)/
+                const vimeoMatch = link.match(vimeoRegExp)
+                const vimeoVideoID = vimeoMatch[3]
+
+                return `https://player.vimeo.com/video/${vimeoVideoID}?color=ffffff&portrait=0`
+
+            default:
+                return ''
+        }
+    } else {
+        return ''
+    }
+}
+
+const VendorProfilePage = (props) => {
   const vendorId = get(props, 'match.params.vendorId', '')
   const vendorProfile = get(props, `data.vendorProfiles.${vendorId}`)
-  return (
+  const vendorPhone = formatPhoneNumber(get(vendorProfile, 'phone', ''))
+  const vendorName = get(vendorProfile, 'name', '')
+  const profileImageUrl = get(vendorProfile, 'photoUrl', defaultImage)
+  const vendorEmail = get(vendorProfile, 'email', '')
+  const vendorBio = get(vendorProfile, 'bio', '')
+  const vendorWebsite = "http://" + get(vendorProfile, 'website', '')
```

Comment:
use a string template here.

Also the user might put `http` in their url. 

If you want to check for something like this you can do

```
const vendorWebsite = get(vendorProfile, 'website', '')
const displayWebsiteUrl = vendorWebsite.indexOf('http:') === -1 ? `http://${vendorWebsite}` : vendorWebsite
```